login/login-page.xml

```diff
@@ -3,16 +3,47 @@
     <ActionBar title="Login" icon="" class="action-bar">
     </ActionBar>
 
-     <StackLayout>
-        <Button text="Log in with Facebook" tap="onLoginFacebookButtonTap"></Button>
-        <Button text="Log in with Google +" tap="onLoginGoogleButtonTap"></Button>
-        <Label text="or"></Label>
+     <StackLayout class="p-t-15">
+        <Button class="btn btn-outline text-left"  
+            tap="onLoginFacebookButtonTap">
+            <FormattedString>
+                <!-- 
+                    To see icons, you need to add font awesome to your project.
+                    Learn more about icon font and how it can be added in this documentation article:
+                    https://docs.nativescript.org/ui/icon-fonts
+                -->
+                <Span text="&#xf09a;" class="fa"></Span>
+                <Span  text="     Log in with Facebook"></Span>
```

Comment:
I find this  uncomfortable. May be we should think for some other solution or leave it without 'padding'?
Try some magic property from the [API](https://docs.nativescript.org/api-reference/classes/_text_span_.span.html) ?

Reply:
The same as above - [here](https://github.com/NativeScript/template-master-detail-kinvey/blob/master/cars/cars-list-page.xml#L43)

I will look for something better in the API :)

Reply:
There is no other way to have offset between two spans in FormattedString, except adding the spaces. If @EmilStoychev  is OK to remove the space between the icon and the text, i will remove it :)